lib/openshift/profile.rb

```diff
@@ -0,0 +1,13 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class Profile < ClusterResource
+    RESOURCE = 'profiles'
+
+    def tuned_profile(user: nil, cached: false, quiet: false)
```

Comment:
@skordas, `cached: true` should be used since this is not a dynamic element, it's a spec.

Reply:
Changed as suggested.